website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -176,4 +176,64 @@ Inline alerts can be added to a section or component or inline with content.
 
 Compact alerts can be added to a section or component or inline with content.
 
-![Placement of compact Alert](/assets/components/alert/alert-placement-compact.png =600x*)
\ No newline at end of file
+![Placement of compact Alert](/assets/components/alert/alert-placement-compact.png =600x*)
+
+## Multiple alerts
+
+In general, displaying multiple alerts on the same page is discouraged as it can negatively impact the page hierarchy and unnecessarily shift emphasis away from the main content on the page.
+
+However, depending on the logic within the application and the type content being expressed to the user, it may be necessary to display more than one Alert on a page. Consider these scenarios and their accompanying guidelines if this is the case.
+
+### Alerts of the same color
+
+!!! Dont
+
+Don’t display more than one of the same `color` Alert on the page. This adds unnecessary visual weight to page and may result in the user being confused about which Alert is more important or which might require attention first.
+
+![Example of multiple Alerts of the same color](/assets/components/alert/multiple-color-alerts-dont.png)
+!!!
+
+!!! Do
+
+Instead, try to combine relevant and overlapping content into a single Alert.
+
+![Example of multiple Alerts combined into one](/assets/components/alert/multiple-color-alerts-do.png)
+
+This may result in the length of the content being longer, but consolidating messages into a single instance of the Alert is a more concise communication pattern.
+!!!
+
+### Ordering of Alerts
+
+Order multiple Alerts by their importance and impact on the user, with `critical` communicating Alerts that must be resolved immediately, and descending in perceived impact to `neutral`.
+
+![Ordering alerts](/assets/components/alert/ordering-alerts.png)
+
+### Alerts of the same type
+
+Try to avoid multiple Alerts of the same type (`page`, `inline`, etc) on the same page.
+
+!!! Do
+
+Instead, use Alerts hierarchically according to their intended prominence and importance to the user.
+
+![An example of multiple types of Alerts on the same page](/assets/components/alert/multiple-alert-types-do.png)
+!!!
```

Comment:
I know it wasn't the intention, but this example makes it seem like we're advocating for critical alerts to always be global. There may be instances, such as within a form, where using an inline alert for critical messages is more appropriate. Is there some way to make this more clear?

Reply:
I've replaced this with a `highlight` Alert to avoid this, but I'll also add some information to the contextual section that might add a bit more clarity